.cirrus.yml

```diff
@@ -319,12 +319,31 @@ task:
         memory: 1G
       package_install_script:
         - apt-get update
-        - apt-get install -y curl chromium
+        - apt-get install -y curl 
+        - apt install -y dbus
         - ln -s -T /usr/bin/chromium /usr/bin/chromium-browser
+        - mkdir -p /var/run/dbus
+        - dbus-daemon --config-file=/usr/share/dbus-1/system.conf --print-address
       env:
         CI_DISTRO: debian
         CI_MAIN_MODULE: /usr/lib/x86_64-linux-gnu/nss/libnssckbi.so
         CI_BAK_MODULE: /usr/lib/x86_64-linux-gnu/nss/libnssckbi.orig.so
+  matrix:
```

Comment:
This matrix only has one entry; I don't think that'll work?  I thiink the only things in this matrix should be the `CI_MAIN_MODULE` and `CI_BAK_MODULE` variables, and there should be two variants (one for CKBI, one for Softoken).

Reply:
Okay! Done.
But the error is still the same

Reply:
Done!
That removes the requirement of repeating the commands above.

Reply:
Looks like the other Chromium test returns the same error but passes (Might be due to Cirrus instructions):
https://cirrus-ci.com/task/6122413401309184
The first error dbus (On the other Chromium test) is fixed hence it returns this:
https://cirrus-ci.com/task/5559463447887872